Library/ViewModels/PledgeShippingLocationViewModel.swift

```diff
@@ -52,17 +52,44 @@ public final class PledgeShippingLocationViewModel: PledgeShippingLocationViewMo
     let selectedLocationId = configData
       .map { $0.3 }
 
-    let shippingShouldBeginLoading = reward
-      .map { $0.shipping.enabled }
-
-    let shippingRulesEvent = Signal.zip(project, reward)
-      .filter { _, reward in reward.shipping.enabled }
-      .switchMap { project, reward -> SignalProducer<Signal<[ShippingRule], ErrorEnvelope>.Event, Never> in
-        AppEnvironment.current.apiService.fetchRewardShippingRules(projectId: project.id, rewardId: reward.id)
+    let shippingShouldBeginLoading = configData
+      .mapConst(true)
+    
+    // CURRENT APP VERSION:
+      
+//    let shippingRulesEvent = Signal.zip(project, reward)
+//      .filter { _, reward in reward.shipping.enabled }
+//      .switchMap { project, reward -> SignalProducer<Signal<[ShippingRule], ErrorEnvelope>.Event, Never> in
+//        AppEnvironment.current.apiService.fetchRewardShippingRules(projectId: project.id, rewardId: reward.id)
+//          .ksr_delay(AppEnvironment.current.apiDelayInterval, on: AppEnvironment.current.scheduler)
+//          .map(ShippingRulesEnvelope.lens.shippingRules.view)
+//          .retry(upTo: 3)
+//          .materialize()
+//      }
```

Comment:
This is the current app implementation that fetches the Shipping Rules for a single reward.